### PR TITLE
remove setup.py and requirements.txt

### PR DESCRIPTION
**Pull request by bjornaer**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -136,4 +136,5 @@ examples/examples
 */hidden.env
 local/
 pipeline-bak/
-*hidden*.env
\ No newline at end of file
+*hidden*.env
+hidden.env
```
Comment (alexpearce) [inline .gitignore:140]: Does `*hidden*.env` not match `hidden.env`?

**Review by alexpearce — APPROVED** [review]
LGTM! 💣

File: .gitignore (hunk 1 above)
Comment (bjornaer) [reply .gitignore:140]: it was not ignoring hidden.env
tbh .gitignore sometimes eludes me

Comment (alexpearce) [reply .gitignore:140]: How about we change this to `*.env`? From what I've seen these are always 'local' files. If examples exist they have different extensions (`*.env.example`) which a dev can copy to `*.env`.

**Pull request merged**